### PR TITLE
[dotnet] Correct product version in http user agent

### PR DESCRIPTION
**Pull request by nvborisenko**
### Description
It puts correct version in http request header like `4.9.1` rather than `4.0.0`. Assembly version is still `4.0.0` which is correct. It is achieved via `AssemblyInformationalVersion` attribute set in bazel.

### Motivation and Context
Highlight correct product version to track usage of selenium.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: dotnet/src/webdriver/Internal/ResourceUtilities.cs
Comment (nvborisenko) [inline dotnet/src/webdriver/Internal/ResourceUtilities.cs:1]: Should we remove unused `AssemblyVersion` method?

Comment (diemol) [reply dotnet/src/webdriver/Internal/ResourceUtilities.cs:1]: Yes, if it is not used, we should remove it.

Comment (nvborisenko) [reply dotnet/src/webdriver/Internal/ResourceUtilities.cs:1]: Removed.

**Review by diemol — APPROVED** [review]
Thank you, @nvborisenko!

**Pull request merged**